java/11/alpine/3.13/detect/6.9.1/buildless/Dockerfile

```diff
@@ -10,7 +10,7 @@ ENV BDS_JAVA_HOME=/usr/lib/jvm/java-11-openjdk
 
 # Download Detect jar
 
-ARG DETECT_SOURCE=https://sig-repo.synopsys.com/bds-integrations-release/com/synopsys/integration/synopsys-detect/6.9.1/synopsys-detect-6.9.1.jar
+ARG DETECT_SOURCE=https://repo.blackduck.com/bds-integrations-release/com/synopsys/integration/synopsys-detect/6.9.1/synopsys-detect-6.9.1.jar
```

Comment:
Since Andrian has created the folder '../com/blackduck/..', do you want to update this path?

Reply:
Since these are older releases, they would be present under com/synopsys/integration and the artifact names would also be unchanged, even in repo.blackduck.com